### PR TITLE
Markdown adjust

### PR DESCRIPTION
**Pull request by KarateCowboy**

### CODE REVIEW

File: src/commands/plugins/update.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ import Plugins from '../../plugins'
 export default class PluginsUpdate extends Command {
   static topic = 'plugins'
   static command = 'update'
-  static description = 'updates installed plugins'
+  static description = 'update a plugin to the latest version'
```
Comment (jdx) [inline src/commands/plugins/update.js]: should say 'update installed plugins'

File: src/commands/update.js [hunk 2]
```diff
@@ -10,6 +10,7 @@ import path from 'path'
 
 export default class Update extends Command {
   static topic = 'update'
+  static description = 'Updates the Heroku CLI'
```
Comment (jdx) [inline src/commands/update.js]: 'update the Heroku CLI'

File: src/commands/which.js [hunk 3]
```diff
@@ -5,6 +5,7 @@ import Plugins from '../plugins'
 
 export default class extends Command {
   static topic = 'which'
+  static description = 'Show the location of a plugin'
```
Comment (jdx) [inline src/commands/which.js]: 'show which plugin a command is from'

File: src/commands/plugins/update.js (hunk 1 above)
Comment (KarateCowboy) [reply src/commands/plugins/update.js]: changed

File: src/commands/update.js (hunk 2 above)
Comment (KarateCowboy) [reply src/commands/update.js]: changed

File: src/commands/which.js (hunk 3 above)
Comment (KarateCowboy) [reply src/commands/which.js]: changed

File: src/commands/plugins/update.js (hunk 1 above)
Comment (jdx) [reply src/commands/plugins/update.js]: I don't see any updates

**Comment by KarateCowboy** [comment]
@dickeyxxx  sorry about that, don't know why they didn't take. `git push origin HEAD:markdown-adjust` about ten times. Took on the last.

**Review by jdx — APPROVED** [review]

**Pull request merged**